### DISCUSSION TITLE
All catppuccin/nvim themes.

### DISCUSSION THREAD

**Discussion by guzman109** [NvChad/NvChad#2494] [Ideas]
Could there be a possibility to add all the Catppuccin themes? Personally I like to keep it simple and switch from latte for light and any of the other 3 for dark in my terminal. I was wondering if this would be added to NVChad to keep everything the same.

I found a configuration file for latte but I would feel more comfortable if all the official Catppuccin themes were to be added to NVChad.

**Comment by siduck** [comment]
no, catppuccin theme is banned in nvchad, even the nvchad's version is old catpuccin which was heavily modified!

**Reply by Tuuuukka** [reply]
> catppuccin theme is banned in nvcha

Why is that though?

**Reply by stddebugprint** [reply]
:clown_face:
